modules/ducktests/tests/ignitetest/services/utils/control_utility.py

```diff
@@ -220,6 +220,96 @@ def snapshot_create(self, snapshot_name: str, timeout_sec: int = 60):
         raise TimeoutError(f'Failed to wait for the snapshot operation to complete: '
                            f'snapshot_name={snapshot_name} in {timeout_sec} seconds.')
 
+    def incremental_snapshot_create(self, snapshot_name: str, exp_increment: int, timeout_sec: int = 60):
```

Comment:
I think it worth to eliminate the dublication in  *incremental_snapshot_create* and *snapshot_create*. Extract common parts. May be use the same function passing the *is_increment* as optional parameter.

Say can we use the --snapshot status for both cases?